catalog/schematron/catalog-schematron-plugin/pom.xml

```diff
@@ -49,6 +49,12 @@
             <groupId>ddf.platform.util</groupId>
             <artifactId>platform-util</artifactId>
         </dependency>
+        <dependency>
```

Comment:
junit should be inherited from the parent by default - you shouldn't have to add this dependency.